macros/edr/dbt_artifacts/dbt_columns/get_dbt_columns_materialization.sql

```diff
@@ -0,0 +1,16 @@
+{% macro get_dbt_columns_materialization() %}
+  {% if var("sync", false) %}
+    {% do return("table") %}
+  {% endif %}
+  {% do return(adapter.dispatch("get_dbt_columns_materialization", "elementary")()) %}
+{% endmacro %}
+
+
+{% macro default__get_dbt_columns_materialization() %}
+  {% do return("view") %}
+{% endmacro %}
+
+
+{% macro bigquery__get_dbt_columns_materialization() %}
+  {% do return("incremental") %}
```

Comment:
Why `incremental` and not `table`?
The `incremental` for our other artifacts is meant so that the `table` materialization won't truncate them and disrupt the caching, but here it's not relevant.

Reply:
Concur on this point. `dbt_columns` don't need to be materialized as incremental as we get the tables info and columns info every time this model is run.